docs/apache-airflow-providers-microsoft-azure/logging/index.rst

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
Missing backward compatibility for links. https://github.com/apache/airflow/pull/17775/files#r693965147